.github/workflows/automatic-prs-from-upstream.yml

```diff
@@ -0,0 +1,24 @@
+name: Create PR from upstream
+on:
+  schedule:
+    - cron: '0 * * * *'
+jobs:
+  create-pr-from-upstream:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Reset to upstream's master
+        run: |
+          git remote add upstream https://github.com/influxdata/telegraf.git
+          git fetch upstream master:upstream-master
+          git reset --hard upstream-master
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.PAT }}
+          title: Upstream changes
+          body: Upstream changes
```

Comment:
I would add some info to the body that the PR is created automatically

Reply:
Done. Please check now.